.github/actions/pip_installation/action.yml

```diff
@@ -31,4 +31,8 @@ runs:
       shell: bash
       run:
         python -m pip install "pytest>=4.5"
+    - name: Install parallel pytest
+      shell: bash
+      run:
+        python -m pip install "pytest-xdist"
```

Comment:
Can you add a line to our README file in [this section](https://github.com/pyccel/pyccel#additional-packages)?

Reply:
Fixed 54de294